es-space-mgmt/main/default/lwc/reservationList/__tests__/reservationList.test.js

```diff
@@ -0,0 +1,231 @@
+/* eslint-disable @lwc/lwc/no-unexpected-wire-adapter-usages */
+import { createElement } from 'lwc';
+import getOpenReservations from '@salesforce/apex/reservationManagerController.getOpenReservations';
+import {
+    registerApexTestWireAdapter,
+    registerTestWireAdapter
+} from '@salesforce/sfdx-lwc-jest';
+import ReservationList from 'c/reservationList';
+import TILE_SELECTION_MC from '@salesforce/messageChannel/Tile_Selection__c';
+import FLOW_STATUS_CHANGE_MC from '@salesforce/messageChannel/Flow_Status_Change__c';
+import { subscribe, MessageContext, publish } from 'lightning/messageService';
+import { refreshApex } from '@salesforce/apex';
+
+// Realistic data with a list of spaces
+const mockOpenReservationsRecords = require('./data/getOpenReservations.json');
+// Register as Apex wire adapter. Some tests verify that provisioned values trigger desired behavior.
+const getOpenReservationsAdapter = registerApexTestWireAdapter(
+    getOpenReservations
+);
+// Register as a standard wire adapter because the component under test requires this adapter.
+// We don't exercise this wire adapter in the tests.
+const messageContextWireAdapter = registerTestWireAdapter(MessageContext);
+
+// mock apex refresh method
+jest.mock(
+    '@salesforce/apex',
+    () => {
+        return {
+            refreshApex: jest.fn()
+        };
+    },
+    { virtual: true }
+);
+
+describe('c-reservation-list', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+        // Prevent data saved on mocks from leaking between tests
+        jest.clearAllMocks();
+    });
+
+    it('getOpenReservations @wire data', () => {
```

Comment:
You're testing too many things in this test (tests need to be unit test). Split those checks in multiple tests.
For example: if you're checking for @wire data, you shouldn't be checking for subscribe calls.
Also try to have a more descriptive test name, it must answer the following question: what's the expected behavior in a given condition?
This could be "displays reservation tiles when there are records"

Reply:
Perhaps it would make sense to create a test suite for @wire tests and another one for LMS test?

Reply:
done! Have the habit of writing integration tests with other frameworks! my bad! have split into proper unit tests. ReReview